pkg/nb/api.go

```diff
@@ -18,7 +18,7 @@ type Client interface {
 	ReadNamespaceResourceOperatorInfoAPI(ReadNamespaceResourceParams) (NamespaceResourceOperatorInfo, error)
 	SetNamespaceStoreInfo(NamespaceStoreInfo) error
 
-	ListAccountsAPI() (ListAccountsReply, error)
```

Comment:
all the list account changes are not needed anymore, can be removed

Reply:
It still needs to be fixed. 
We can divide it into a different PR, but I see no point in that. 
I prefer living it here, not even divide to commits.
WDYT?

Reply:
What is the issue that needs to be fixed?
The ListAccountsParams is just an empty struct, why do we need to pass it as params? look at listBucketAPI, there are no params there too. Also, I couldn't find a place in the operator code that uses this function.

Reply:
the operator code is not using this RPC call, that's true, but this is still an issue in this call. 
If the code is there he might work.
I can think of 2 things:
1. Fix the not working calls for RPC even if they are not being used.
2. Remove those calls completely.

I don't think leaving a call that is not working is the correct way.

Regarding this specific call, It expects to get an empty object as a param, and this is what I have added 😃

Reply:
@liranmauda, what is not working with the current method signature? I also don't see a reason to add `ListAccountsParams`. can you elaborate on the need?

Reply:
Ok, so I see that it's not matching the account_api. in any case, I suggest doing it in a different PR and providing all other possible params (filter, etc.)

Reply:
Opened #872 and will remove this code